src/plugin.js

```diff
@@ -308,9 +308,21 @@ const eme = function(options = {}) {
   const player = this;
 
   const emeError = emeErrorHandler(player);
+  let contentHasEnded = false;
 
   player.ready(() => onPlayerReady(player, emeError));
 
+  player.on('ended', () =>{
+    contentHasEnded = true;
+
+    player.one('play', () => {
+      if (contentHasEnded && player.currentTime() === 0) {
+        contentHasEnded = true;
```

Comment:
I think we might want to set `contentHasEnded` back to false here? Also do we need the `currentTime() === 0` check above?

Reply:
Actually, I was doing some more testing and turns out we don't need this `play` listener on `ended` at all. 
An `encrypted` event is triggered on replay or seek-back after content has ended. So, setupSessions and handleEncryptedEvent are called automatically [here](https://github.com/videojs/videojs-contrib-eme/blob/main/src/plugin.js#L263)